lookout/style/format/rules.py

```diff
@@ -57,11 +56,16 @@ class Rules(BaseEstimator, ClassifierMixin):
 
     CompiledRulesType = Dict[int, CompiledFeatureRules]
 
+    BottomUpGreedyBudget = NamedTuple("BottomUpGreedyBudget", (
+        ("absolute", bool), ("value", Union[float, int])))
+
     log = logging.getLogger("Rules")
 
     def __init__(self,
                  base_model: Union[DecisionTreeClassifier, RandomForestClassifier],
-                 prune_branches=True, prune_attributes=True, uncertain_attributes=True):
+                 prune_branches=True, prune_branches_algorithm="top-down-greedy",
```

Comment:
I think we will have other pruning methods and it is better to create a namespace for its names, like:
```python
class PruneAlgorithms:
    top_down_greedy = "top-down-greedy"
```

Reply:
So far we have only two, let's YAGNI for now

Reply:
I will add third one :)

Reply:
but it is possible that for different scenarios, trees or data different methods can be more suitable. Or do I miss something?

Reply:
It is not clear now, but once we run experiments and see which pruning works the best (and in sane time) - we remove the rest.

Reply:
I didn't touch that @zurk since it was still debated. I also agree with @vmarkovtsev that once we'll have tested the 3 approaches a clear winner should emerge.